massa-api/src/api.rs

```diff
@@ -51,6 +55,16 @@ impl MassaApiServer for API<ApiV2> {
         Ok(self.0.version)
     }
 
+    fn get_best_parents(
+        &self,
+        page_request: Option<PageRequest>,
```

Comment:
In API V2, we should anticipate and avoid breaking changes, we should create a struct ApiRequest to wrap all existing and coming parameters

Reply:
Done

Reply:
I forgot to update openrpc.json, i'll do it

Reply:
done